pages/landing/enterprise-eks/pricing/index.html

```diff
@@ -8,69 +8,143 @@
 permalink: /eks-for-enterprise/pricing/
 
 categories:
-  - title: Feature
+  - title: EKS
+    features:
+      - name: Control plane
+        description: Support for configuring, deploying, and managing the EKS control plane.
+        plans: ["Standard", "Enterprise"]
+      - name: Worker nodes
+        description: "Support for deploying any of the three worker node types: self-managed ASG, EKS Managed Node Groups, and Fargate."
+        plans: ["Standard", "Enterprise"]
+      - name: Core services
+        description: Suite of core Kubernetes applications every EKS cluster should have, including AWS LoadBalancer controller, external-dns, cluster-autoscaler, and more.
+        plans: ["Standard", "Enterprise"]
+      - name: CI/CD
```

Comment:
Nit: perhaps "Basic CI/CD" to make it clearer there is an "advanced" version below, and to be consistent with the other "basic" and "advanced" options?